flight/targets/revolution/fw/Makefile

```diff
@@ -245,7 +246,7 @@ CDEFS += -DUSE_$(BOARD)
 ifeq ($(ENABLE_DEBUG_CONSOLE), YES)
 CDEFS += -DPIOS_INCLUDE_DEBUG_CONSOLE
 endif
-CDEFS += -DSTM32F40_41xxx
+CDEFS += -DSTM32F40_41xxx -DSTM32F405xx
```

Comment:
Perhaps you can just define STM32F405xx... check CMSIS and StdPeriph to see what it does re: the legacy STM32F40_41xxx.